pydantic_ai_slim/pydantic_ai/agent.py

```diff
@@ -616,7 +617,7 @@ async def main():
 
         # TODO: Instead of this, copy the function tools to ensure they don't share current_retry state between agent
         #  runs. Requires some changes to `Tool` to make them copyable though.
-        for v in self._function_tools.values():
+        for v in self._get_tools().values():
```

Comment:
Can you please change this to `_get_function_tools` to continue to distinguish them from output tools?